website/content/en/pre-docs/AWS/constraints.md

```diff
@@ -0,0 +1,158 @@
+---
+title: "Provisioning Configuration"

```

Comment:
Does it make more sense to call this page AWS Provider? And name the file AWS/provider.md

Reply:
I think "AWS Provider" is comperably accurate, however I want to give some indication as to what this page helps users do, which (from my perspective) is "configure the instances/nodes provisioned by the AWS provider", or "provisioning configuration"